diff --git a/node_modules/@medusajs/dashboard/dist/app.js b/node_modules/@medusajs/dashboard/dist/app.js
index 630384f..7ccbba6 100644
--- a/node_modules/@medusajs/dashboard/dist/app.js
+++ b/node_modules/@medusajs/dashboard/dist/app.js
@@ -127315,7 +127315,7 @@ var init_variant_media_section = __esm({
     VariantMediaSection = ({ variant }) => {
       const { t: t5 } = (0, import_react_i18next122.useTranslation)();
       const media = (variant.images || []).filter(
-        (image) => image.variants?.some((variant2) => variant2.id === variant2.id)
+        (image) => image.variants?.some((variant2) => variant2.id === variant.id)
       );
       return /* @__PURE__ */ (0, import_jsx_runtime176.jsxs)(import_ui131.Container, { className: "divide-y p-0", children: [
         /* @__PURE__ */ (0, import_jsx_runtime176.jsxs)("div", { className: "flex items-center justify-between px-6 py-4", children: [
@@ -127926,10 +127926,10 @@ var init_edit_product_variant_media_form = __esm({
       const { handleSuccess } = useRouteModal();
       const allProductImages = variant.product?.images || [];
       const allVariantImages = (variant.images || []).filter(
-        (image) => image.variants?.some((variant2) => variant2.id === variant2.id)
+        (image) => image.variants?.some((variant2) => variant2.id === variant.id)
       );
       const unassociatedImages = allProductImages.filter(
-        (image) => !image.variants?.some((variant2) => variant2.id === variant2.id)
+        (image) => !image.variants?.some((variant2) => variant2.id === variant.id)
       );
       const [selection, setSelection] = (0, import_react136.useState)({});
       const [isSidebarOpen, setIsSidebarOpen] = (0, import_react136.useState)(false);
diff --git a/node_modules/@medusajs/dashboard/dist/product-variant-detail-N7YAKUOH.mjs b/node_modules/@medusajs/dashboard/dist/product-variant-detail-N7YAKUOH.mjs
index b951400..327aa06 100644
--- a/node_modules/@medusajs/dashboard/dist/product-variant-detail-N7YAKUOH.mjs
+++ b/node_modules/@medusajs/dashboard/dist/product-variant-detail-N7YAKUOH.mjs
@@ -383,7 +383,7 @@ import { jsx as jsx6, jsxs as jsxs3 } from "react/jsx-runtime";
 var VariantMediaSection = ({ variant }) => {
   const { t } = useTranslation5();
   const media = (variant.images || []).filter(
-    (image) => image.variants?.some((variant2) => variant2.id === variant2.id)
+    (image) => image.variants?.some((variant2) => variant2.id === variant.id)
   );
   return /* @__PURE__ */ jsxs3(Container3, { className: "divide-y p-0", children: [
     /* @__PURE__ */ jsxs3("div", { className: "flex items-center justify-between px-6 py-4", children: [
diff --git a/node_modules/@medusajs/dashboard/dist/product-variant-media-IOAZGVL2.mjs b/node_modules/@medusajs/dashboard/dist/product-variant-media-IOAZGVL2.mjs
index 30ea30e..6d800f9 100644
--- a/node_modules/@medusajs/dashboard/dist/product-variant-media-IOAZGVL2.mjs
+++ b/node_modules/@medusajs/dashboard/dist/product-variant-media-IOAZGVL2.mjs
@@ -42,10 +42,10 @@ var EditProductVariantMediaForm = ({
   const { handleSuccess } = useRouteModal();
   const allProductImages = variant.product?.images || [];
   const allVariantImages = (variant.images || []).filter(
-    (image) => image.variants?.some((variant2) => variant2.id === variant2.id)
+    (image) => image.variants?.some((variant2) => variant2.id === variant.id)
   );
   const unassociatedImages = allProductImages.filter(
-    (image) => !image.variants?.some((variant2) => variant2.id === variant2.id)
+    (image) => !image.variants?.some((variant2) => variant2.id === variant.id)
   );
   const [selection, setSelection] = useState({});
   const [isSidebarOpen, setIsSidebarOpen] = useState(false);
diff --git a/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-detail/components/variant-media-section/variant-media-section.tsx b/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-detail/components/variant-media-section/variant-media-section.tsx
index 2b68c21..bb9d3bd 100644
--- a/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-detail/components/variant-media-section/variant-media-section.tsx
+++ b/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-detail/components/variant-media-section/variant-media-section.tsx
@@ -14,7 +14,7 @@ export const VariantMediaSection = ({ variant }: VariantMediaSectionProps) => {
 
   // show only variant scoped images
   const media = (variant.images || []).filter((image) =>
-    image.variants?.some((variant) => variant.id === variant.id)
+    image.variants?.some((imageVariant) => imageVariant.id === variant.id)
   )
 
   return (
diff --git a/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-media/components/edit-product-variant-media-form/edit-product-variant-media-form.tsx b/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-media/components/edit-product-variant-media-form/edit-product-variant-media-form.tsx
index 167d2f2..2b66c98 100644
--- a/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-media/components/edit-product-variant-media-form/edit-product-variant-media-form.tsx
+++ b/node_modules/@medusajs/dashboard/src/routes/product-variants/product-variant-media/components/edit-product-variant-media-form/edit-product-variant-media-form.tsx
@@ -44,11 +44,11 @@ export const EditProductVariantMediaForm = ({
 
   const allProductImages = variant.product?.images || []
   const allVariantImages = (variant.images || []).filter((image) =>
-    image.variants?.some((variant) => variant.id === variant.id)
+    image.variants?.some((imageVariant) => imageVariant.id === variant.id)
   )
 
   const unassociatedImages = allProductImages.filter(
-    (image) => !image.variants?.some((variant) => variant.id === variant.id)
+    (image) => !image.variants?.some((imageVariant) => imageVariant.id === variant.id)
   )
 
   const [selection, setSelection] = useState<Record<string, true>>({})
